styles/components/_menu.scss

```diff
@@ -1,60 +1,69 @@
 .page__container {
-  width: 100vw;
-  height: calc(100vh - 2rem);
-  background-color: white;
-  text-align: center;
+  z-index: -1;
+  height: 100vh;
+  background: url(/static/images/ui/background.png) no-repeat center;
+  background-size: cover;
 }
 
 .menu-window {
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  padding: 60px 0 60px 60px;
   &__header {
-    position: relative;
-    display: flex;
-    flex-direction: column;
-    padding: 32px;
-  }
-  &__heading {
-  }
-  &__description {
+    font-family: Roboto;
+    font-weight: 700;
   }
 }
 
 .menu {
-  position: relative;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 32px;
-  &__graphic {
-    &-container {
-      width: 30vw;
-      height: 50vh;
-      background-color: cadetblue;
-    }
-  }
   &__buttons {
     display: flex;
+    width: 20%;
     flex-direction: column;
-  }
-  &__button {
-    &--rules {
+    margin-bottom: 55vh;
+    text-align: start;
+    &__button {
+      border: none;
+      margin-top: 50px;
+      background: none;
+      color: #000;
+      font-family: Roboto;
+      font-size: 24px;
+      font-weight: 700;
+      outline: none;
+      text-align: start;
+      &:hover {
+        color: #f00;
+        cursor: pointer;
+      }
+    }
+
+    &--start {
       position: absolute;
-      right: 2rem;
-      bottom: 0;
-      width: 2rem;
-      height: 2rem;
+      top: 75vh;
+      left: 40vw;
+      width: 25%;
+      height: 13%;
```

Comment:
perhaps try a different approach with the units and centering. The word is not centered right now.